src/components/ProjectDetailPage/__tests__/ProjectDetailPage.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { render } from '@testing-library/react';
+import { ProjectDetailPage } from '../ProjectDetailPage';
+import { MemoryRouter } from 'react-router-dom';
+
+jest.mock('../../../services/projectService');
+
+describe('display ProjectDetailPage component', () => {
+  test('renders ProjectDetail', () => {
+    render(
+      <MemoryRouter>
+        <ProjectDetailPage />
```

Comment:
Hi Jakub,
Since you have already gotten the mock data from mockProjectService, you should also verify that the data were displayed properly by getting some element or text from the screen to see if it exists or match with your mock data. See my last review about how to implement some tests over the elements on the page.